### ISSUE TITLE
OpenAir: get rid of annoying warning on repeated point

### ISSUE THREAD

**Issue by alus-** [alus-it/AirspaceConverter#167]
In OpenAir the last point (DP) should match the first one to "close" the airspace.
But can happen that there is a geometry (for example an arc, DA) at the end of the airspace and then still a point DP added, matching the last point of the last geometry, to match as well the first point DP to properly close the geometry.
This very special situation will generate a warning about a repeated point, but it is not a repeated point it is just an assertion that the airspace is so closed...
Of course if the last DP of previous example is than repeated this must result as repeated point warning.
So basically, the last DP matching the first DP if also equal to the end of the last geometry must not generate a repeated point warning.

**Comment by alus-** [comment]
Fixed!
